Assets/MixedRealityToolkit.Tests/PlayModeTests/PlayModeTestUtilities.cs

```diff
@@ -1,23 +1,82 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
+#if !WINDOWS_UWP
+// When the .NET scripting backend is enabled and C# projects are built
```

Comment:
Do the test assemblies need to be included in the build, or can we mark them as editor-only to potentially work around this issue?

Reply:
@wiwei  did you try to just modify the .asmdef files to only include these projects in the editor? Was there an issue that you ran into that prevented you from doing this? Reading your comment it seems like there was a build issue when building for UWP .NET backend. I assumed I needed to do the same ifdef here.

Reply:
For now I'll check this in as is and maybe we can solve this more elegantly with an asmdef after we hear from will.